openedx/core/djangoapps/content_libraries/tests/base.py

```diff
@@ -131,7 +130,7 @@ def _create_library(
             # We're not actually using this value any more, but we're keeping it
             # in the API testing for backwards compatibility for just a little
             # longer. TODO: Remove this once the frontend stops sending it.
-            "collection_uuid": uuid.uuid4(),
+            # "collection_uuid": uuid.uuid4(),
```

Comment:
If this change works with frontend-app-library-authoring, then let's just delete this line and the comments above.